cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils_test.go

```diff
@@ -513,20 +514,20 @@ func TestParseMemoryCapacity(t *testing.T) {
 		expectedQuantity: zeroQuantity.DeepCopy(),
 		expectedError:    true,
 	}, {
-		description:      "valid quantity without unit type",
+		description:      "valid quantity",
 		annotations:      map[string]string{memoryKey: "456"},
 		expectedError:    false,
-		expectedQuantity: resource.MustParse("456Mi"),
+		expectedQuantity: *resource.NewQuantity(456*units.MiB, resource.DecimalSI),
```

Comment:
i'm finding this a little bit confusing, do the providers multiply the value if it has no unit type?

Reply:
The providers assume that the values they read don't have units and that the integer represents the `Mi` value, so we create a new quantity here that is matching that assumption

Reply:
what you are saying makes sense, i'm just a little confused by the mechanics here since i thought the annotation will say "456", so the multiplication seems odd to me.

Reply:
Providers must set and integer which represent Mebibytes https://github.com/openshift/kubernetes-autoscaler/pull/146/files#diff-5a3aabbbd7e4ff27771547853ef4f2eeR189

Here we convert that to bytes https://github.com/openshift/kubernetes-autoscaler/pull/146/files#diff-5a3aabbbd7e4ff27771547853ef4f2eeR197